docs/getting-started/adding-graphback-to-your-project.md

```diff
@@ -117,6 +117,20 @@ const { typeDefs, resolvers, contextCreator } = buildGraphbackAPI(schema, {
 });
 ```
 
+Graphback automatically creates a `CRUDService` with default configuration values. You can override this with the `serviceCreator` option:
+
+```ts
+import { buildGraphbackAPI, createCRUDService } from 'graphback';
+import { RedisPubSub } from 'graphql-redis-subscriptions';
+
+const { typeDefs, resolvers, contextCreator } = buildGraphbackAPI(schema, {
+  // highlight-start
+  serviceCreator: createCRUDService({ pubSub: new RedisPubSub() }),
+  // highlight-end
+  dataProviderCreator: knexDbProviderCreator
```

Comment:
Can we add the missing import?

Reply:
I will redact that part to keep the focus purely on the service creator.